src/modules/75-cf/FFCustomMicroFrontendProps.types.ts

```diff
@@ -55,7 +55,7 @@ export interface FFCustomMicroFrontendProps {
   customComponents: {
     ContainerSpinner: typeof ContainerSpinner
     Description: typeof Description
-    EvaluationModal: typeof EvaluationModal
+    PolicyManagementEvaluationModal: typeof PolicyManagementEvaluationModal
```

Comment:
As mentioned, this is a breaking change in this form for MFEs, if you alias as the previous name, it'll give the MFE owners a chance to adjust
```suggestion
    EvaluationModal: typeof PolicyManagementEvaluationModal
```